### DISCUSSION TITLE
Aurora Construct parameterGroup - MySQL 8

### DISCUSSION THREAD

**Discussion by ricardoheyn** [aws/aws-cdk#17958] [Q&A] [answered]
Hello,

How can I specify the Aurora (MySQL version 8) properly in the ParameterGroup Prop?

I am receiving the following:

`CREATE_FAILED        | AWS::RDS::DBCluster                         | Database/Db () The Parameter Group default.aurora-mysql5.7 with DBParameterGroupFamily aurora-mysql5.7 cannot be used for this instance. Please use a Parameter Group with DBParameterGroupFamily aurora-mysql8.0 (Service: AmazonRDS; Status Code: 400; Error Code: InvalidParameterCombination
`

I can see listed here: `aws rds describe-db-engine-versions --query "DBEngineVersions[].DBParameterGroupFamily" --engine aurora-mysql`
but looks like my change `auroraEngineVersion: rds.AuroraMysqlEngineVersion.of("8.0.mysql_aurora.3.01.0")` is not enough.

Thank you in advance.

**Comment by skinny85** [comment, marked as answer]
You probably need to create your own `ParameterGroup`, with engine equal to `rds.AuroraMysqlEngineVersion.of("8.0.mysql_aurora.3.01.0")`, and pass it when creating your Cluster.

BTW, I think you're missing the second parameter to the `AuroraMysqlEngineVersion.of()` call - the major version of the engine, which should be `'8.0'` in your case (the default is `'5.7'`).

The default Parameter Group for Aurora MySQL is `'default.aurora-mysql5.7'`, as the error message says.
